model/src/test/kotlin/utils/PurlUtilsTest.kt

```diff
@@ -48,6 +52,23 @@ class PurlUtilsTest : StringSpec({
             }
         }
     }
+
+    "The purl test cases should never have an empty namespace" {
```

Comment:
Would it make sense to turn these tests into require conditions, to make them more separate from real tests?

Reply:
While I agree that these tests are special in the sense that they do not test our code, but perform sanity checks on the test suite data, I'm not sure if it makes things better to use require conditions. How exactly would that look like? Would you keep the actual test cases, but just swap out the assertions, e.g. `shouldNot beEmpty()`, with an equivalent `require()` statement?

Reply:
I had in mind doing this when test cases are read. For example, add an `val testCases = ....also {  // require checks here }

Reply:
I'll consider a follow-up PR.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/9340.